extensions/sql-migration/src/dialog/validationResults/validateIrDialog.ts

```diff
@@ -447,6 +456,16 @@ export class ValidateIrDialog {
 		}
 	}
 
+	private _formatError(error: Error): Error {
+		if (error?.message?.startsWith(HttpStatusCodes.GatewayTimeout)) {
+			return {
```

Comment:
same for this function

Reply:
it could be reused, but I would wait to refactor it when it can be reused.